monday_lecture

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
```

Comment:
It's very nice you create your personal notes, but we shouldn't include it in github repo - save it somewhere in your local machine but delete it from repo ;)